pyuvsim/tests/test_analyticbeam.py

```diff
@@ -8,6 +8,7 @@
 import numpy as np
 import nose.tools as nt
 from scipy.special import j1
+from scipy.interpolate import UnivariateSpline
```

Comment:
I don't see where you're using this import

Reply:
That was for the old version of the test. Thanks for catching it.